language/stdlib/modules/libra_system.mvir

```diff
@@ -24,7 +24,18 @@ module LibraSystem {
 
     resource BlockMetadata {
       // Height of the current block
+      // TODO: Should we keep the height?
       height: u64,
+
+      // current real world time provided by consensus.
+      timestamp: u64,
```

Comment:
We should say what this means (UNIX time?) and perhaps add a TODO for wrapping it in a better type (e.g., `Time.T` that supports comparison but not equality).